decompiler/extractor/main.cpp

```diff
@@ -23,12 +26,18 @@ enum class ExtractorErrorCode {
   DECOMPILATION_GENERIC_ERROR = 4030
 };
 
+enum GameIsoFlags { FLAG_JAK1_BLACK_LABEL = (1 << 0) };
+
```

Comment:
Is there documentation for these flags somewhere?  I'm not really understanding what it's being used for

Reply:
The flag is used in `compile` below to set the value of `*jak1-full-game*` which the build system (and the game) needs to know since black label is missing the `COMMONJ.SBK` file.

Reply:
`*jak1-full-game*` is a boolean, not a bit string.